ocaml/parsing/jane_syntax.mli

```diff
@@ -276,6 +276,19 @@ module Layouts : sig
     (Jkind.annotation * Parsetree.attributes) option
 end
 
+module Instances : sig
+  (** The name of an instance module. Gets converted to [Global.Name.t] in the
+      flambda-backend compiler. *)
+  type instance =
+    { head : string;
+      args : (instance * instance) list
```

Comment:
Can we try to distinguish the parameter and value in `instance * instance`?

In particular, the parameter cannot have arguments.